contracts/src/Teleporter/upgrades/README.md

```diff
@@ -137,18 +127,67 @@ Using specific version:
         ITeleporterMessenger teleporterMessenger = _getTeleporterMessenger();
 ```
 
-#### Receiving from specific Teleporter versions
+### Receiving from specific Teleporter versions
 
 `TeleporterUpgradeable` also provides an initial implementation of [ITeleporterReceiver.receiveTeleporterMessage](../ITeleporterReceiver.sol) that ensures `_msgSender` is a `TeleporterMessenger` contract with a version greater than or equal to `minTeleporterVersion`. This supports the case where a dApp wants to upgrade to a new version of the `TeleporterMessenger` contract, but still wants to be able to receive messages from the old Teleporter contract.The dApp can override `_receiveTeleporterMessage` to implement its own logic for receiving messages from Teleporter contracts.
 
-#### Pausing Teleporter version interactions
+## Managing a TeleporterUpgradeable dApp
 
-Dapps that inherit from `TeleporterUpgradeable` can pause Teleporter interactions by calling `TeleporterUpgradeable.pauseTeleporterAddress`. This function prevents the contract from interacting with the paused Teleporter address when sending or receiving Teleporter messages.
```

Comment:
nit: Perhaps add a sentence here to summarize what managing a TeleporterUpgradable dApp involves (updating versions, pausing interactions, etc.). Otherwise we would have a subsection heading directly following a section heading.

Reply:
Good call, added now